content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,239 @@
+---
+title: April '21 Community Gems
+date: 2021-04-30
+description: |
+  A roundup of technical Q&A's from the [DVC](#dvc) and [CML](#cml) community. 
```

Comment:
small correction: I think links (and other basic MD) is supported in the `descriptionLong` only